<launch>
    <rosparam command="load" file="$(find fast_lio2_eigen)/config/mid360.yaml" />
    <node pkg="fast_lio2_eigen" type="ieskf_frontend_node" name="ieskf_frontend_node" output="screen" />
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find fast_lio2_eigen)/rviz/show.rviz" />
</launch>